utils/python/CIME/case.py

```diff
@@ -411,6 +412,24 @@ def configure(self, compset_name, grid_name, machine_name=None,
         for key, value in pes_rootpe.items():
             mach_pes_obj.set_value(key,int(value))
 
+        for compclass in self._component_classes:
+            if compclass == "DRV":
+                continue
+            key = "NINST_%s"%compclass
+            mach_pes_obj.set_value(key, ninst)
+            key = "NTASKS_%s"%compclass
+            if key not in pes_ntasks.keys():
+                mach_pes_obj.set_value(key,1)
+            key = "NTHRDS_%s"%compclass
+            if compclass not in pes_nthrds.keys():
+                mach_pes_obj.set_value(compclass,1)
+
+        # FIXME - this is a short term fix for dealing with the restriction that
```

Comment:
This is a weird quirk - one version of our land ice model, cism1, is serial.   We haven't figured out a better place to handle this.